adopt_net0/components/technologies/genericTechnologies/stor.py

```diff
@@ -61,7 +61,7 @@ class Stor(Technology):
       .. math::
         E_{t} = E_{t-1} * (1 - \\lambda_1) - \\lambda_2(\\Theta) * E_{t-1} + {\\eta}_{in} * Input_{t} - 1 / {\\eta}_{out} * Output_{t}
 
-    - If ``allow_only_one_direction == 1``, then only input or output can be unequal
+    - If ``allow_only_one_direction_precise == 1``, then only input or output can be unequal
```

Comment:
This should only be allow_only_one direction, right? Maybe we should have a brief docu on what the precise does?